### PR TITLE
fix: sort languages correctly-by numbers not strings

### PR DESCRIPTION
**Pull request by MatousJobanek**
this PR fixes languages sorting - originally it sorted the languages by strings which didn't work in case of bigger numbers, eg:
```
Go : 123
Makefile: 23
```
as a result the Makefile was prior to the Go because the first number is bigger.
The cause of this mistake: a combination of lack of my brain activity with an attempt to avoid problems with duplicated ints/keys :-)

### CODE REVIEW

File: pkg/git/langs.go [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
 	}
-	return sortLanguages(contentSizes, reversedMap)
+	sort.Ints(contentSizes)
+
+	var sortedLangs []string
+	for i := len(contentSizes) - 1; i >= 0; i-- {
+		sortedLangs = append(sortedLangs, reversedMap[contentSizes[i]]...)
+	}
+	return sortedLangs
 }
```
Comment (xcoulon) [inline pkg/git/langs.go]: Have you considered using the [sort.Sort](https://golang.org/pkg/sort/) interface? Maybe you could declare a custom type to sort the languages in reverse order of their size? (just an idea, though)

Comment (MatousJobanek) [reply pkg/git/langs.go]: good idea. do you mean something like this: 5eb83c2 ?

Comment (xcoulon) [reply pkg/git/langs.go]: yes, that looks simpler with one less loop in the code ;)

**Pull request merged**